GoogleUtilitiesComponents.podspec

```diff
@@ -0,0 +1,39 @@
+Pod::Spec.new do |s|
+  s.name             = 'GoogleUtilitiesComponents'
+  s.version          = '1.0.0'
+  s.summary          = 'Google Utilities Component Container for Apple platforms.'
+
+  s.description      = <<-DESC
+An internal Google utility that is a dependency injection system for libraries to depend on other
+libraries in a type safe and potentially weak manner.
+Not intended for direct public usage.
+                       DESC
+
+  s.homepage         = 'https://github.com/firebase/firebase-ios-sdk/tree/master/GoogleUtilitiesComponents'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'UtilitiesComponents-' + s.version.to_s
+  }
+
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.11'
+  s.tvos.deployment_target = '10.0'
+
+  s.cocoapods_version = '>= 1.4.0'
+  s.prefix_header_file = false
```

Comment:
We might want `s.static_frameworks = true` at least until CocoaPods 1.9.x releases and gives fine grain control to Podfiles for controlling static vs dynamic.

Reply:
Good call, will add.